tests/TestCase/Routing/RouterTest.php

```diff
@@ -2260,6 +2260,40 @@ public function testParseRoutePath(): void
             'action' => 'view',
         ];
         $this->assertSame($expected, Router::parseRoutePath('Vendor/Cms.Management/Admin/Articles::view'));
+
+        $expected = [
+            'controller' => 'Bookmarks',
+            'action' => 'view',
+            '1',
+        ];
+        $this->assertSame($expected, Router::parseRoutePath('Bookmarks::view/1'));
+        $this->assertSame($expected, Router::parseRoutePath('Bookmarks::view/1/'));
+
+        $expected = [
+            'plugin' => 'Cms',
+            'controller' => 'Articles',
+            'action' => 'edit',
+            '2023',
+            '03',
+            '5th',
+        ];
+        $this->assertSame($expected, Router::parseRoutePath('Cms.Articles::edit/2023/03/5th'));
+
+        $expected = [
+            'controller' => 'Bookmarks',
+            'action' => 'view',
+            'organisation' => 'cakephp',
+            'repository' => 'debug_kit',
+        ];
+        $this->assertSame($expected, Router::parseRoutePath('Bookmarks::view/organisation:cakephp/repository:debug_kit'));
+
+        $expected = [
+            'controller' => 'Bookmarks',
+            'action' => 'view',
+            'organisation' => 'cakephp',
```

Comment:
Nitpick: should be z since we use AE

Reply:
my german speaking brain is just getting in my ways sometimes 😁